modules/obfuscating-dvo-data.adoc

```diff
@@ -7,7 +7,11 @@
 [id="obfuscating-deployment-validation-operator-data_{context}"]
 = Obfuscating Deployment Validation Operator data
 
-Cluster administrators can configure the Insights Operator to obfuscate data from the Deployment Validation Operator (DVO), if the Operator is installed. When the `workload_names` value is added to the *insights-config* `ConfigMap` object, workload names—rather than UIDs—are displayed in Insights for Openshift, making them more recognizable for cluster administrators. 
+By default, when you install the Deployment Validation Operator (DVO), the name and UID of a resource are included in the data that is captured and processed by the Insights Operator for {product-title}. 
```

Comment:
```suggestion
By default, when you install the Deployment Validation Operator (DVO), the name and user ID (UID) of a resource are included in the data that is captured and processed by the Insights Operator for {product-title}. 
```

Reply:
In this case the UID refers to the `unique identifier` not the `user ID`

Reply:
@dfitzmau @opokornyy - Thank you both.

Changed line 10 to:
By default, when you install the Deployment Validation Operator (DVO), the name and unique identifier (UID) of a resource are included in the data that is captured and processed by the Insights Operator for {product-title}.